backend/tracim_backend/views/core_api/user_controller.py

```diff
@@ -720,6 +724,29 @@ def open_message_stream(self, context, request: TracimRequest, hapic_data=None)
             headerlist=headers, charset="utf-8", status_code=200, body=stream_opened_event
         )
 
+    @hapic.with_api_doc(tags=[SWAGGER_TAG__USER_CONFIG_ENDPOINTS])
+    @check_right(has_personal_access)
+    @hapic.input_path(UserIdPathSchema())
+    @hapic.output_body(UserConfigSchema())
+    def get_user_config(self, context, request: TracimRequest, hapic_data: HapicData) -> Dict:
+        """
+        get all the configuration parameters for the given user
+        """
+        config_api = UserConfigApi(user=request.candidate_user, session=request.dbsession)
```

Comment:
This should be `request.current_user`, `candidate_user` is the not authenticated/validated user. It shouldn't have any effect here, only good practice to use `current_user`.

Reply:
I thought `current_user` was the one that makes the request and `candidate_user` the one given in the URL? current_user would therefore be wrong, no?

Reply:
… you're right, sorry.

Reply:
No problem :-)